docs/concepts/workloads/controllers/deployment.md

```diff
@@ -553,7 +553,7 @@ the Deployment will not have any effect as long as the Deployment is paused.
 
 Eventually, resume the Deployment and observe a new ReplicaSet coming up with all the new updates:
 ```shell
-$ kubectl rollout resume deploy nginx
+$ kubectl rollout resume deploy/nginx
```

Comment:
`kubectl rollout resume deploy/nginx` and `kubectl rollout resume deploy nginx` both work. 

However from this PR I found inconsistency in this doc. Deployment name should be "nginx-deployment" instead of "nginx". @zs1379 would you fix all the incorrect references in this doc while you're at it?